### ISSUE TITLE
"connection show" shows the password

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbemtools#397]
Example:
```
$ pywbemcli connection show testconn
name: testconn (current)
  server: http://localhost
  default-namespace: root/cimv2
  user: usr1
  password: blah
  timeout: 30
  verify: True
  certfile: None
  keyfile: None
  mock-server: 
  ca-certs: ['/etc/pki/ca-trust/extracted/openssl/ca-bundle.trust.crt', '/etc/ssl/certs', '/etc/ssl/certificates']
```

That has the risk that someone who demonstrates pywbemcli unintentionally reveals the passsword of a server.

**Proposal:**

Change the default to not show the password, but allow showing it when explicitly requested, e.g. as follows:
* Add a boolean command option `--show-password` to "connection show".
* If the option is not set, show the password value as 8 asterisks.

**Discussion point:** Which release?

**Comment by KSchopmeyer** [comment]
I suggest this release.   This one is straight forward and with the option, It would be ok to put it on the connection list also.

**Comment by andy-maier** [comment]
Addressed by PR #398.
